contentcuration/contentcuration/frontend/channelList/views/KCard/index.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+
+  <BaseCard
+    :title="title"
+    :headingLevel="headingLevel"
+    :titleLines="titleLines"
+  >
+    <div>
+      <KImg
+        v-if="layout === 'vertical' && thumbnailDisplay !== 'none'"
+        :src="thumbnailSrc"
+        :height="300"
+        :width="500"
+        :isDecorative="false"
+        altText="thumbnail image"
+        :appearanceOverrides="{
+          objectFit: thumbnailScaleType,
+          backgroundColor: $themeTokens.fineLine
+        }"
+        :style="KImgVerticalSmallStyle"
+      />
+      <slot v-if="!thumbnailSrc" name="thumbnailPlaceholder"></slot>
+
+      <div class="horizontal-layout-style">
+
+        <KImg
+          v-if="layout === 'horizontal' && thumbnailDisplay === 'large'"
+          :src="thumbnailSrc"
+          :height="300"
+          :width="600"
+          :isDecorative="false"
+          altText="thumbnail image"
+          :appearanceOverrides="{
+            objectFit: thumbnailScaleType,
+            backgroundColor: $themeTokens.fineLine
+          }"
+        />
+        <slot v-if="!thumbnailSrc" name="thumbnailPlaceholder"></slot>
+
+        <div>
+          <KRouterLink
+            :to="to"
+            :appearanceOverrides="{
+              'text-decoration': 'none',
+              'cursor': 'pointer',
+            }"
+            tabindex="-1"
+          >
+            <div
+              v-if="thumbnailDisplay === 'none'"
+              class="spacing"
+            >
+              <slot name="aboveTitle"></slot>
+              <slot name="title"></slot>
+              <slot name="belowTitle"></slot>
+            </div>
+            <KGrid
+              v-else
+              tabindex="-1"
```

Comment:
What was the reason for setting tabindex here? I would expect `KGrid` to not be focusable by default.